Automattic/wp-calypso
# Framework: Cleanup unused legacy connected applications store and methods

We've reduxified the connected applications, and it's time to remove the old store and unused undocumented methods. This is what this PR takes care of. This is the last PR of the connected applications reduxification sprint.

For more context of the full connected applications reduxification effort, you can see this PR: #24175.

To test:
* Checkout this branch.
* Verify that the removed store and methods are no longer imported/used at all.
* Smoke test connected applications a bit just in case
